.github/2.0/cookbooks/1vs2.md

```diff
@@ -51,16 +101,65 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
    with f:
        f.index(random_docs(10000))
 ```
+
+
+<table>
+<tr>
+<td>
+<b>2.0.0rc1</b>
+</td>
+</tr>
+<tr>
+<td>
+
 ```console
 ✅ done in ⏱ 8 seconds 🐎 1194.1/s
 ```
-- 1.2.4
+
+
+</td>
+</tr>
+<tr>
+<td>
+<b>1.4.2</b>
+</td>
+</tr>
+<tr>
+<td>
+
 ```console
 ✅ done in ⏱ 8 seconds 🐎 1127.9/s
 ```
 
+
+</td>
+</tr>
+</table>
+
 </details>
 
-1. Keyword 3
-1. Keyword 4
-1. Keyword 5
\ No newline at end of file
+## 2.0 Gives YOU the Power
+
+Jina 1.0 was a complex beast with lots of boilerplate code and not so much transparency. In 2.0 we're putting the power back in your hands. Want to write your own `.logger`, `.save()`, `.load()` or context manager interface for your Executor? Now that power is yours.
+
+## 2.0 is More Pythonic
+
+Jina 2.0 embraces explicit above implicit: No more Drivers. You can manipulate Documents in custom Executors directly with all the power that entails:
+
+```python
+from jina import Document, DocumentArray, Executor, requests
+
+class MyExecutor(Executor):
+    @requests
+    def add_world(self, docs: DocumentArray):
+        for doc in docs:
+            doc.text += 'world'
+```
+
+## 2.0 is Lighter
+
+Jina has gone on a diet for 2.0. We've dramatically cut the layers of abstraction, removing Drivers, Peas and Pods to give you a simpler, more elegant Flow experience. 
```

Comment:
removing Drivers and hiding Peas and Pods may be better?

Reply:
Yes, this should definitely be changed.

Reply:
I worded it as "de-emphasizing Peas and Pods". We talk a lot about being more transparent elsewhere, so "hiding" goes against that